### PR TITLE
Example cms

### PR DESCRIPTION
**Pull request by cedricserugendo**
modification of the names of the example CMs for more clarity

### CODE REVIEW

File: cm/README.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ Base directory for the calculation modules (cm).
 A calculation module is an asynchronous slow task (from seconds to hours of latency for the answer). 
 A calculation module takes a set of rasters, a selection vector shape and an optional set of parameters
 
+For basic example, see [example_empty](./example_empty) CM. 
+And for more detailed example, see [example_multiply](./example_multiply) CM.
```
Comment (melhk) [inline cm/README.md]: I would write "For a non implemented example..." (and add comments in the code to indicate the parts to be completed),  and "And for a simple example..."

File: cm/example_multiply/README.md [hunk 2]
```diff
@@ -16,10 +23,10 @@ Once EnerMaps launched, the frontend should be available on this adress : http:/
 
 ## Inputs 
 
-To make the CM work, we need thres inputs:
+To make the CM work, we need three inputs:
 * the raster we want to studied,
 * the areas we want to studied,
```
Comment (melhk) [inline cm/example_multiply/README.md]: "...we want to study"

File: cm/example_empty/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Remarks
+
+The *example_empty* is empty CM.
+
+It only serves as an example for patners developing others CM.
+
+For more detailed CM, see [*example_multiply*](../example_multiply) CM.
```
Comment (melhk) [inline cm/example_empty/README.md]: "For an example of implementation"

File: cm/example_empty/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Remarks
+
+The *example_empty* is empty CM.
+
+It only serves as an example for patners developing others CM.
```
Comment (melhk) [inline cm/example_empty/README.md]: The *example_empty" is the generic CM structure to be implemented.

File: cm/README.md (hunk 1 above)
Comment (cedricserugendo) [reply cm/README.md]: I made the suggested changes

File: cm/example_multiply/README.md (hunk 2 above)
Comment (cedricserugendo) [reply cm/example_multiply/README.md]: I made the suggested changes

File: cm/example_empty/README.md (hunk 3 above)
Comment (cedricserugendo) [reply cm/example_empty/README.md]: I made the suggested changes

File: cm/example_empty/README.md (hunk 4 above)
Comment (cedricserugendo) [reply cm/example_empty/README.md]: I made the suggested changes

File: cm/README.md (hunk 1 above)
Comment (melhk) [reply cm/README.md]: Y cannot see your changes, do you have to do another merge request? (I'm not used to work  whit github yet)

Comment (cedricserugendo) [reply cm/README.md]: I think you should go to the "Files changed" section to see the latest version of the branch. 
Unfortunately, I think we don't have the update in the conversation section, we just have the "outdated" label.

File: cm/example_multiply/README.md (hunk 2 above)
Comment (melhk) [reply cm/example_multiply/README.md]: I wanted to say:

To make the CM work, we need three inputs:
* the raster we want to study,
* the areas we want to study,

File: cm/example_multiply/README.md [hunk 5]
```diff
@@ -1,3 +1,10 @@
+# Preliminary remarks
+
+The *example_multiply* is complete CM.
+It serves as an example for patners developing others CM.
+
+For more basic CM, see [*example_empty*](../example_empty) CM.
```
Comment (melhk) [inline cm/example_multiply/README.md]: Make these changes consistent with those made in "cm/example_empty/README.md".

**Review by melhk — COMMENTED** [review]
Some last comments

File: cm/README.md (hunk 1 above)
Comment (melhk) [reply cm/README.md]: Thank you!

File: cm/example_multiply/README.md (hunk 2 above)
Comment (cedricserugendo) [reply cm/example_multiply/README.md]: sorry, I fixed the error in the last commit

File: cm/example_multiply/README.md (hunk 5 above)
Comment (cedricserugendo) [reply cm/example_multiply/README.md]: I made the change in the last commit

**Review by melhk — APPROVED** [review]
Modifications are ok

**Pull request merged**